src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopies = [];
+  const stateCopy = { ...state };
```

Comment:
it can be an empty object
```suggestion
  const stateCopy = {};
```

Reply:
I've tried with an empty object, but the tests don't go through as the copy doesn't have starting values of the state object.